solidity-plugin/src/main/java/org/sonarsource/solidity/SoliditySensor.java

```diff
@@ -126,8 +126,8 @@ private static void saveFileMeasures(SensorContext context, FileMeasures fileMea
     context.<Integer>newMeasure().on(inputFile).withValue(fileMeasures.getStatementNumber()).forMetric(CoreMetrics.STATEMENTS).save();
     context.<Integer>newMeasure().on(inputFile).withValue(fileMeasures.getFileCognitiveComplexity()).forMetric(CoreMetrics.COGNITIVE_COMPLEXITY).save();
     context.<Integer>newMeasure().on(inputFile).withValue(fileMeasures.getFileComplexity()).forMetric(CoreMetrics.COMPLEXITY).save();
-    context.<Integer>newMeasure().on(inputFile).withValue(fileMeasures.getContractComplexity()).forMetric(CoreMetrics.COMPLEXITY_IN_CLASSES).save();
-    context.<Integer>newMeasure().on(inputFile).withValue(fileMeasures.getFunctionComplexity()).forMetric(CoreMetrics.COMPLEXITY_IN_FUNCTIONS).save();
+    // context.<Integer>newMeasure().on(inputFile).withValue(fileMeasures.getContractComplexity()).forMetric(CoreMetrics.COMPLEXITY_IN_CLASSES).save();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=20&issues=AWQdvLOs_bEzQ-kUUpnI&open=AWQdvLOs_bEzQ-kUUpnI)

[ISSUE_KEY]: AWQdvLOs_bEzQ-kUUpnI